seed/static/seed/partials/analyses.html

```diff
@@ -0,0 +1,60 @@
+<div class="page_header_container">
+    <div class="page_header">
+        <div class="left page_action_container">
+            <a ui-sref="organizations" class="page_action"><i class="fa fa-chevron-left"></i> {$:: 'Organizations' | translate $}</a>
+        </div>
+        <div class="page_title">
+            <h1>{$:: org.name $}</h1>
+        </div>
+        <div class="right page_action_container">
+        </div>
+    </div>
+</div>
+<div class="section_nav_container">
+    <div class="section_nav" ng-include="::urls.static_url + 'seed/partials/accounts_nav.html'"></div>
+</div>
+<div class="section">
+    <div class="section_header_container">
+        <div class="section_header fixed_height_short">
+            <div class="left section_action_container">
+            	<h2><i class="fa fa-user"></i> {$:: 'Analyses' | translate $}</h2>
+           	</div>
+        </div>
+    </div>
+        <div class="section_content_container">
+        <div class="section_content">
+            <div class="table_list_container">
+                <table id="analyses-table" class="table table-striped table_highlight_first">
+                    <thead>
+                        <tr>
+                            <th translate>Analysis Name (User Defined)</th>
+                            <th translate>Actions</th>
+                            <th translate>Number of Properties</th>
+                            <th translate>Type</th>
+                            <th translate>Run Status</th>
+                            <th translate>Run Date</th>
+                            <th translate>Run Duration</th>
+                            <th translate>Run Author</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-repeat="analysis in analyses  | filter:filter_params:strict">
+                            <td>{$:: analysis.name $}</td>
+                            <td>
+                            	<i class="glyphicon glyphicon-pencil" aria-hidden="true"></i>
+                            	<i class="glyphicon glyphicon-trash" aria-hidden="true"></i>
+                            </td>
+                            <td>{$:: analysis.number_of_analysis_property_views $}</td>
+                            <td>{$:: analysis.service $}</td>
+                            <td class="analysis-status {$:: analysis.status.toLowerCase() $}">{$:: analysis.status $}</td>
+                            <td>{$:: analysis.start_time | date : 'MM-dd-yyyy' $}</td>
+                            <td>{$:: (analysis.end_time - analysis.start_time < 1 ? '' : analysis.end_time - analysis.start_time) | date : 'MM-dd-yyyy' $}</td>
+                            <td>{$:: analysis.user $}</td>
```

Comment:
<img width="1008" alt="Screen Shot 2020-12-08 at 12 19 59 AM" src="https://user-images.githubusercontent.com/30608004/101454180-c47f1700-38ed-11eb-99a2-a2f9a99c747e.png">
3 things:

- Can we add the time to the start time?
- I'm not sure what's happening with the duration value.
- Author is ID right now. We should probably make that first and last name. I'm thinking that we should GET the org users and use that response to get the name.

Reply:
Added some filters that should cover the last two.